sljit_src/sljitNativePPC_common.c

```diff
@@ -181,9 +184,12 @@ static const sljit_u8 freg_map[SLJIT_NUMBER_OF_FLOAT_REGISTERS + 3] = {
 #define FRSP		(HI(63) | LO(12))
 #define FSUB		(HI(63) | LO(20))
 #define FSUBS		(HI(59) | LO(20))
-#define LD		(HI(58) | 0)
+#define LD		(HI(58))
```

Comment:
Why `| 0` is removed? As far as I remember the last two bits define the instruction types for 64 bit load/stores, and that 0 wants to represent it.